temboardui/plugins/statements/__init__.py

```diff
@@ -384,7 +384,7 @@ def getstatdata_sample(request, mode, start, end, dbid=None):
     query = (
         select(cols)
         .select_from(subquery)
-        .where(c.calls is not None)
+        .where(c.calls != 0)
```

Comment:
Thinking more about this, the query would now also return rows with `calls` being `NULL`.
Is this what we want? If not, perhaps `.where(c.calls)` would cover filtering out both `NULL` and `0` values.

Reply:
`calls` should actually never be `NULL`.
https://github.com/dalibo/temboard/blob/master/temboardui/model/alembic/versions/statements.sql#L66